hpx/components/security/certificate.hpp

```diff
@@ -119,7 +120,7 @@ namespace hpx { namespace components { namespace security
 #endif
 }}}
 
-namespace boost { namespace serialization
+namespace hpx { namespace traits {
 {
```

Comment:
Braces here look wrong. The built should be tried with security enabled, but that might trigger latent bugs.